src/main/c/org_broadinstitute_hellbender_utils_bwa_BwaMemIndex.c

```diff
@@ -4,6 +4,19 @@
 #include "jnibwa.h"
 #include "bwa/bwa_commit.h"
 
+#define jStringToConstChar(A,B) \
```

Comment:
I'm dropping this macro all together; the ```Get``` and ```Release``` should occur before and after the libbwa call so needs to be split in two anyway.